remove-png-icon.patch

```diff
@@ -0,0 +1,26 @@
+From 89eabc9913cce12ff0516339bd3594c9420c4f3c Mon Sep 17 00:00:00 2001
+From: bbhtt <62639087+bbhtt@users.noreply.github.com>
+Date: Sun, 31 Jul 2022 11:44:22 +0530
+Subject: [PATCH] Stop installing png
```

Comment:
why?

Reply:
Because of wrong dimensions of the png icon https://github.com/flathub/mx.pwmc.Svgvi/pull/3#issuecomment-1200356060

Reply:
ah ok.

you could just remove it in post-install instead.

Reply:
Added a post install command to remove it.